README.md

```diff
@@ -136,7 +153,7 @@ The name of the variable is `projectile-rails-keymap-prefix`.
 To attach the `projectile-rails` keymap to the `projectiles` keymap one can do:
 
 ```el
-(setq projectile-rails-keymap-prefix (kbd "C-c p C-r"))
+(setq projectile-rails-keymap-prefix (kbd "C-c r"))
 ```
```

Comment:
I wonder if this change makes sense given the text above (`to attach to the projectiles keymap`).

I'd probably remove the entire block or whatever makes sense.